rosbag2/test/rosbag2/mock_storage.hpp

```diff
@@ -38,6 +38,7 @@ class MockStorage : public rosbag2_storage::storage_interfaces::ReadWriteInterfa
   MOCK_METHOD0(get_all_topics_and_types, std::vector<rosbag2_storage::TopicMetadata>());
   MOCK_METHOD0(get_metadata, rosbag2_storage::BagMetadata());
   MOCK_CONST_METHOD0(get_bagfile_size, uint64_t());
+  MOCK_METHOD0(split_database, void());
```

Comment:
Not needed?

Reply:
whoops, that should be in the next PR. Removing since it's not needed here.